src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
```

Comment:
Could you please undo the formatting changes?

Reply:
(I think there was a change in VS at some point that started doing this. Might be worth letting it do that once as a separate change to avoid everyone having to revert it if they make resx edits in VS.)

Reply:
@jkotas any real concern here? I think the difference is really getting rid of the spaces at the end of the lines which I think it is a good thing to do. Also, you can control the review not showing space differences if this cause a problem while reviewing the code. please let me know if you still want me revert the spaces.

Reply:
Visual Studio insists on inserting these spaces. I have verified that it is the case with the latest VS 2019. Next time somebody opens and edits this file in VS, the spaces will be back.

There is a lot of adding and removing spaces in the history of these resource files, e.g. https://github.com/dotnet/coreclr/commit/286c43a716fd6382c1cbc730dcdd9b24705dde48 .

The majority of these files in corefx repo do have the spaces and majority of people seems to edit these files in VS and thus prefer the VS format, so it is what we should stick to.

Reply:
I have reverted that and have used VS 2019 in the editing the resources.